### DISCUSSION TITLE
“command not found: create-next-app” how do I create a next application?

### DISCUSSION THREAD

**Discussion by AWAIS97** [vercel/next.js#19298] [Help]
I am new to next.js and following the tutorial.
I am using this command to create my app
```npx create-next-app```
but it is giving an error which is

Error: EPERM: operation not permitted, mkdir 'C:\Users\Awais' command not found: create-next-app

how to create the app? p.s I have nodejs installed.

**Comment by smith-the-row** [comment]
install the create-next-app package worked for me 
though not the best solution 
but it's the available

**Comment by JasmineLaiHua** [comment]
First you need to install React (global):
npm install -g react next react-router

Then try to install create-next-app package:
npm install -g create-next-app

Then you can create nextjs app again on any folder: 
npx create-next-app my-app

I have been had the same problem and these command lines worked for me. Hope my suggestion can help you!

**Reply by helloromero** [reply]
Thanks! that worked for me.

**Reply by constRJ** [reply]
This also works for me, thanks for this @JasmineLaiHua

**Reply by lubishak** [reply]
thanks! it worked

**Reply by HarshUpadhyay27** [reply]
thank you so much

**Comment by shambashib20** [comment]
I basically use another method.
firstly I create the project folder on which I want to work

``mkdir testproj``

Then, I access the folder using

``cd testproj``

Then I use 

``npm init next-app <application name which you want to use>``

That's it! The scaffold gets ready. Hope this works for you!
